partcad-cli/src/partcad_cli/click/commands/init.py

```
@@ -14,19 +14,19 @@
 
 @click.command(help="Create a new PartCAD package in the current directory")
 # TODO-95: All long options everywhere
-@click.option("-p", is_flag=True, help="Initialize this package as private")
+@click.option("-p", "--private", is_flag=True, help="Initialize this package as private")
 @click.pass_context
-def cli(ctx: click.rich_context.RichContext, p):
+def cli(ctx: click.rich_context.RichContext, private):
```

Ruff (0.8.2)

Missing return type annotation for public function `cli`

Add return type annotation: `None`

(ANN201)